lib/libzpool/zrepl_mgmt.c

```diff
@@ -545,10 +545,10 @@ uzfs_zvol_destroy_snapshot_clone(zvol_state_t *zv, zvol_state_t *snap_zv,
 	    clone_zv->zv_name, clonename, zv->zv_name);
 
 	/* Destroy clone's snapshot */
-	ret = uzfs_destroy_internal_all_snap(clone_zv);
+	ret = uzfs_destroy_all_internal_snapshots(clone_zv);
 	if (ret != 0) {
```

Comment:
I think we should break if ret is nonzero. Any reason why we are not breaking from here?

Reply:
I meant - return if ret is nonzero

Reply:
yes. it should return to caller if `uzfs_destroy_all_internal_snapshots` return non-zero.